src/helpers/collaborator-check.ts

```
@@ -0,0 +1,161 @@
+    logger.debug("No issue found in payload, skipping collaborator check.");
+    return false;
+  }
+
+  const owner = payload.repository.owner.login;
+  const repo = payload.repository.name;
+  const issueNumber = issue.number;
+
+  // Build list of users to exclude (sender + reward recipients)
+  const excludeLogins = new Set<string>([payload.sender.login]);
+  if (rewardRecipients) {
+    rewardRecipients.forEach(login => excludeLogins.add(login));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)